web/index.html

```diff
@@ -74,6 +78,9 @@
 
           core.locationProvider = locationProvider;
           core.startNavigation(route, config);
+
+          document.getElementsByClassName("controls-start")[0].style.display = "none";
+          document.getElementsByClassName("controls-stop")[0].style.display = "block";
```

Comment:
This is used to hide one and unhide another.

Reply:
Oh! I see now; the `block` distracted me 😂 I think it's a bit more idiomatic to use `style.removeProperty("display")`: https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty

Reply:
It would be like:
```
document.getElementById("controls-start").style.setProperty("display", "none");
document.getElementById("controls-stop").style.removeProperty("display");

document.getElementById("controls-start").style.removeProperty("display");
document.getElementById("controls-stop").style.setProperty("display", "none");
```

I feel like this is a little more confusing...

Reply:
Ahh, I see. Yes, it's a bit ugly with the really long method invocation.

My main goal is to make it clear that what's going on is a property toggle between two states: overridden to none, and the default.

Maybe this is an improvement? Another solution is to set the display property to an empty string, but that's completely absurd IMO ;) `removeProperty` makes it clear that you're going back to the original value.

```javascript
document.getElementById("controls-start").style.display = "none";
document.getElementById("controls-stop").style.removeProperty("display");

document.getElementById("controls-start").style.removeProperty("display");
document.getElementById("controls-stop").style = "display";
```

Reply:
I tested, `removeProperty` doesn't work here. We cannot use DOM API to manipulate styles defined in .css files and we can only add new styles on the top of it to override. 🤔 Looks like we'll have to keep it that way.

Reply:
Oh, wow; that's wild! I learned something new today 😂